### DISCUSSION TITLE
Local port calls since v13 upgrade

### DISCUSSION THREAD

**Discussion by jaksenko** [vercel/next.js#52630] [Help] [answered]
### Summary

Since we upgraded to v13 our NewRelic dashboards show weird external services calls which we are unable to identify. This is the big picture (2nd deployment was v13 upgrade):
![Screenshot 2023-07-13 at 10 49 26](https://github.com/vercel/next.js/assets/87317406/7e9867d5-e570-4a80-9156-3d2ea7b1e22b)
and this is the list of external services it calls:
![Screenshot 2023-07-13 at 10 50 44](https://github.com/vercel/next.js/assets/87317406/2d1ac011-0c7a-4ef9-99e2-f30dd5686e25)
You can see they are all local ports (except for sentry but that's what we account for), but they all seem random. We are 100% sure it is connected to Next.js being upgraded from v12 to v13, but as I scanned through changelog I was unable to find anything related to it.

Does anybody have any idea what this is about?

**Comment by icyJoseph** [comment, marked as answer]
Could it be this? https://github.com/vercel/next.js/discussions/51691#discussioncomment-6341657?
